src/player/ui/components.rs

```diff
@@ -116,5 +116,10 @@ pub fn controls(width: usize) -> String {
     let len: usize = controls.concat().iter().map(|x| x.len()).sum();
     let controls = controls.map(|x| format!("{}{}", x[0].bold(), x[1]));
 
-    controls.join(&" ".repeat((width - len) / (controls.len() - 1)))
+    let mut controls = controls.join(&" ".repeat((width - len) / (controls.len() - 1)));
+    controls.push_str(match width % 2 {
```

Comment:
This is a bit hacky, is it possible to achieve the same result just by tweaking the division operation on the previous line?

If not, then a comment would be useful to explain why this is necessary.

Reply:
Tweaking the division on the previous line didn't work, and this was the least hacky solution I could come up with.